hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -74,6 +71,15 @@
 
 import scala.collection.JavaConversions;
 
+
+import static org.apache.hudi.AvroConversionUtils.createRdd;
```

Comment:
same static imports question style question, has this changed recently ?

Reply:
nothing changed. Added static import to avoid multiple reference in code.